utils/build-script

```diff
@@ -2018,6 +2027,11 @@ iterations with -O",
     parser.add_argument(
         "--enable-tsan-runtime",
         help="enable Thread Sanitizer on the swift runtime")
+    parser.add_argument(
+        "--enable-lsan",
+        help="enable Leaks Sanitizer for swift tools",
```

Comment:
@fjricci To be honest, if this is an issue, I think I need to also make a small LLVM change since I called the flag their Leaks as well. Your thoughts?